index.js

```diff
@@ -0,0 +1,35 @@
+import { DateTime } from 'luxon';
+import showSection from './modules/navigation.js';
+import bookDB from './modules/bookDbConstruct.js';
+
+document.addEventListener('DOMContentLoaded', () => {
+  const listBtn = document.querySelector('.list-section');
+  const addBtn = document.querySelector('.add-section');
+  const contactBtn = document.querySelector('.contact-section');
+  const clockDiv = document.querySelector('.clock');
+
+  listBtn.addEventListener('click', () => showSection('list'));
+  addBtn.addEventListener('click', () => showSection('form'));
+  contactBtn.addEventListener('click', () => showSection('contact'));
+
+  showSection('list');
+
+  function updateClock() {
+    const now = DateTime.now().toLocaleString(DateTime.DATETIME_FULL);
+    clockDiv.textContent = now;
+  }
```

Comment:
- [ ] Kindly change this to an arrow function in keeping with the ES6 syntax requirement.